### ISSUE TITLE
Panel must not be registered by default

### ISSUE THREAD

**Issue by sebastienheyd** [santigarcor/laratrust#406]
Hello,

In the default configuration file, I think "panel.register" must be set to false by default.

Someone who is going to update the package will automatically end up with a permission management page.

Anyone who knows that the application is using the package will know that it is then possible to access it, and thus change user permissions.

However the page returns a mix error.

**Comment by santigarcor** [comment]
Makes sense. BTW I'll add the auth middleware by default, and about the mix error, did you publish the assets using `php artisan vendor:publish --tag=laratrust-assets --force`?

**Comment by santigarcor** [comment]
c175ef07ba4a2e8e9c5ed87353e85109c57543fa fixes this.

**Comment by sebastienheyd** [comment]
No problem with the assets, I didn't publish them voluntarily.

I developed a boilerplate with Adminlte3 that uses Laratrust, I already have a permissions control panel. 

It's while upgrading that I saw in the documentation that there was this panel provided, while testing I could have edited the permissions of the people already in the database (well, only if I had published the assets ;) ). 

It's nice to see a well maintained package, I've been using it for three years now, thanks for the work done !
